x/ccv/provider/keeper/genesis.go

```diff
@@ -22,39 +23,146 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 		}
 	}
 
+	k.SetValidatorSetUpdateId(ctx, genState.ValsetUpdateId)
+	for _, v2h := range genState.ValsetUpdateIdToHeight {
+		k.SetValsetUpdateBlockHeight(ctx, v2h.ValsetUpdateId, v2h.Height)
+	}
+
+	for _, cccp := range genState.ConsumerAdditionProposals {
+		// prevent implicit memory aliasing
+		cccp := cccp
+		if err := k.SetPendingConsumerAdditionProp(ctx, &cccp); err != nil {
```

Comment:
Probably we don't want `SetPendingConsumerAdditionPro` to return an error.  @MSalopek we should be consistent re. using `cdc.Marshal` and `cdc.MustMarshal`. Similarly with unmarshal.